spec/models/post_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.new(author_id: 1, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
+  end
+  before { subject.save }
+
+  it 'should have a title' do
+    subject.title = nil
+    expect(subject).to be_invalid
+
+    subject.title = 'Bruno'
+    expect(subject).to be_valid
+  end
+
+  it 'should not be too long' do
+    subject.title = 'b' * 251
+    expect(subject).to be_invalid
+  end
+
+  it 'comments_counter should be an integer greater than or equals to 0' do
+    subject.comments_counter = nil
+    expect(subject).to be_invalid
+
+    subject.comments_counter = -1
+    expect(subject).to be_invalid
+
+    expect(subject.comments_counter).to be_integer
+
+    subject.comments_counter = 3
+    expect(subject).to be_valid
+  end
+
+  it 'likes_counter should be an integer greater than or equals to 0' do
+    subject.likes_counter = nil
+    expect(subject).to be_invalid
+
+    subject.likes_counter = -1
```

Comment:
- [ ]  Great job working on the requirements but I think you still have some falling tests so I highly recommend solving them since Testing plays a vital role in the development of robust and reliable software. In the context of Ruby on Rails applications, having a solid testing strategy can prevent bugs, improve maintainability, and provide developers with the confidence to make changes and add new features without introducing regressions. so please work on them to be solved, As requested by the previous reviewer ✅✅

- Here's the error messages I face when running the tests ⬇
![image](https://github.com/kambereBr/Publisher/assets/58553711/45b3783c-f8ff-4646-a58c-00a554ffa061)
![image](https://github.com/kambereBr/Publisher/assets/58553711/b7692407-9be9-4cf7-b23f-8d35feb8dc05)

Reply:
Thanks, dear @Reem-lab for your review 👏 

Now, the issue is fixed!